core/src/main/java/com/homeaway/streamplatform/streamregistry/health/StreamRegistryHealthCheck.java

```diff
@@ -136,10 +138,10 @@ private synchronized void setConsumerRegistrationHealthy(boolean value) {
     protected synchronized Result check() {
         try {
             validateCreateStream();
-            validateStateStore();
             validateKStreamState();
             validateProducerRegistration();
             validateConsumerRegistration();
+            validateStateStore();
```

Comment:
@neoword 
moved the StreamLookupValidation to the end so that GlobalKTable has enough time to pick the newly inserted stream. (avoided having any sleep statments)